### PR TITLE
(feat) doc about migrating from taiga docker

### PR DESCRIPTION
**Pull request by yamila-moreno**
This PR documents the migration from taiga5 docker to taiga6 docker

- [x] create a taiga5 installation from https://github.com/taigaio/docker-taiga-5
- [x] run first with `populate-db` to have an admin/123123 user
- [x] enter the back container and run sample_data
- [x] check you have the environment of 5 up and running
- [x] build and compile the documentation locally
- [x] where the documentation says "branch" stable, go to "taiga-6"
- [x] You'll probably need to `docker pull taigaio/taiga-front:alpha`
- [x] follow the instructions ("RECOMMENDED INSTALLATION")
- [x] check they are clear
- [x] you end up with a complete environment from taiga6
- [x] merge this PR
- [x] close [this task](https://tree.taiga.io/project/taiganext/task/1177)

### CODE REVIEW

**Comment by tdelatorre** [comment]
Typo:
"Exit the container and get **the get** the dump file"

File: upgrades-5to6.adoc [hunk 1]
```diff
@@ -41,9 +59,47 @@ $ pg_dump -U $TAIGA_DB_USER -h $TAIGA_DB_HOST $TAIGA_DB_NAME > taiga5_db.sql
...
+[source,bash]
+----
+:/# cd
+:/# pg_dump -U $TAIGA_DB_USER -h $TAIGA_DB_HOST $TAIGA_DB_NAME > taiga5_db.sql
+----
+
+Exit the container and get the dump file:
+[source,bash]
+----
+$ docker cp {CONTAINER_NAME}:/root/taiga5_db.sql .
```
Comment (tdelatorre) [inline upgrades-5to6.adoc:89]: Delete /root

File: upgrades-5to6.adoc [hunk 2]
```diff
@@ -142,13 +197,6 @@ server {
 ----
```
Comment (tdelatorre) [inline upgrades-5to6.adoc:197]: Add: "exit the container"

File: upgrades-5to6.adoc (hunk 1 above)
Comment (yamila-moreno) [reply upgrades-5to6.adoc]: fixed!

File: upgrades-5to6.adoc (hunk 2 above)
Comment (yamila-moreno) [reply upgrades-5to6.adoc:199]: fixed!

**Pull request merged**